polkadot/node/core/pvf/common/src/worker/mod.rs

```diff
@@ -71,18 +100,39 @@ macro_rules! decl_worker_main {
 				},
 			}
 
+			let mut worker_dir_path = None;
 			let mut node_version = None;
-			let mut socket_path: &str = "";
+			let mut can_enable_landlock = false;
+			let mut can_unshare_user_namespace_and_change_root = false;
 
-			for i in (2..args.len()).step_by(2) {
+			let mut i = 2;
```

Comment:
Yes, and also because it didn't seem necessary. This isn't a user-facing binary so we don't need nice output and all the other features. Also... compile times.